chart/infra-server/templates/argo/secrets.yaml

```diff
@@ -12,7 +12,7 @@ data:
     artifactRepository:
       archiveLogs: true
       gcs:
-        bucket: argo-artifacts-development
+        bucket: rhacs-infra-artifacts
```

Comment:
Can we make this a secret? This is a public repo and knowledge of the bucket name can aid attackers to access things...

Reply:
Added to https://issues.redhat.com/browse/ROX-24351